x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -275,6 +279,49 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
                 </EuiFlexGroup>
               </EuiFlexItem>
             </EuiFlexGroup>
+            {!dissmissAlertErrors &&
+            alert.executionStatus.status === AlertExecutionStatusValues[2] ? (
```

Comment:
**nit:** it doesn't read clearly to me that it's filtering for errors. If `AlertExecutionStatusValues[2]` is replaced with `error` I think it will work just as fine and the linter will fail if ever `error` isn't a possible value for `alert.executionStatus.status`.

```suggestion
            alert.executionStatus.status === 'error' ? (
```